src/pages/publication/PublicationFormTabs.tsx

```diff
@@ -52,7 +41,7 @@ export const PublicationFormTabs: FC<PublicationFormTabsProps> = ({ handleTabCha
       <LinkTab
         label={`2. ${t('heading.references')}`}
         {...a11yProps('references')}
-        error={hasTouchedError(errors, touched, referenceFieldNames)}
+        error={hasTouchedError(errors, touched, publicationFieldNames)}
```

Comment:
tilbake til naming: her vil vi kun ha feil i felter i reference tab, men nå virker det som vi sjekker for feil i hele publication form

Reply:
ja, så her gir det også mening å endre tilbake til referenceFieldNames